docs/get-started/table-theme-options.qmd

```diff
@@ -0,0 +1,203 @@
+---
+title: Table theme options
+jupyter: python3
+---
+
+Great Tables exposes options to customize the appearance of tables via two methods:
+
+* [](`~great_tables.GT.tab_style`) - targeted styles (e.g. color a specific cell of data).
+* [](`~great_tables.GT.tab_options`) - broad styles (e.g. color the header and source notes).
+
+Both methods target parts of the table, as shown in the diagram below.
+
+![](/assets/gt_parts_of_a_table.svg)
+
+This page covers how to style and theme your table using `GT.tab_options()`,
+which is meant to quickly set a broad range of styles.
+In the future, even more granular options will become available via `GT.tab_style()`.
+
+```{python}
+from great_tables import GT, exibble
+
+gt_ex = (GT(exibble.head(5), rowname_col="row", groupname_col="group")
+    .tab_header("THE HEADING", "(a subtitle)")
+    .tab_stubhead("THE STUBHEAD")
+    .tab_source_note("THE SOURCE NOTE")
+)
+```
+
+## Table option parts
+
+As the graph above showed, tables are made of many parts---such as the heading, column labels, and stub. `.tab_options()` organizes options based on table part.
+
+The code below illustrates the table parts `.tab_options()` can target, by setting the background color for various parts.
+
+
+```{python}
+(
+    gt_ex
+    .tab_options(
+        container_width = "100%",
+        table_background_color="lightblue",
+        heading_background_color = "gold",
+        column_labels_background_color="aquamarine",
+        row_group_background_color="lightyellow",
+        stub_background_color="lightgreen",
+        source_notes_background_color="#f1e2af",
+    )
+)
+```
+
+Notice two important pieces:
+
+* The argument `heading_background_color="gold"` sets the heading part's background to gold.
+* Parts like `container` and `table` are the broadest. They cover all the other parts of the table.
+
+
+## Finding options: part, type, attribute
+
+Option names follow this format:
+
+```python
+{part name}_{type}_{attribute}
+```
+
+For example, the option `row_group_border_top_color` has these pieces:
+
+* **part**: `row_group`
+* **type**: `border_top`
+* **attribute**: `color`
+
+:::{.callout-note}
+Here are the parts supported in `.tab_options()`:
+
+* container, table
+* heading, source_note
+* column_labels, row_group, stub, stub_row
+* table_body
+:::
+
+## Styling borders
+
+Many table parts support customizing border colors and style.
+This is shown below for column labels.
+
+```{python}
```

Comment:
I couldn't see the result of setting these options. Is there something else I need to set?!

Reply:
Use this instead:

```py
gt_ex.tab_options(
    heading_border_bottom_color="blue",
    heading_border_bottom_style="solid"
)
```

The reason the other settings may not work is because the shared border (bottom of header, and top of column labels) has a rule where the larger sized part wins.

Reply:
Like, if you use:

```python
gt_ex.tab_options(
    column_labels_border_top_color="blue",
    column_labels_border_top_style="solid",
    column_labels_border_top_width="5px",
)
```

Then you'll see the blue border because the `5px` size is beating out the size of the `heading_border_bottom...` border.

Reply:
Ah, thanks! I might use this, and then mention the interaction of borders, if that's okay!